api/src/database/migrations/20211103B-update-special-geometry.ts

```diff
@@ -1,8 +1,11 @@
 import { Knex } from 'knex';
+// @ts-ignore
+import Client_Oracledb from 'knex/lib/dialects/oracledb';
 
 export async function up(knex: Knex): Promise<void> {
+	const specialKey = knex.client instanceof Client_Oracledb ? '"special"' : 'special';
 	await knex('directus_fields')
-		.update({ special: knex.raw(`REPLACE(special, 'geometry,', 'geometry.')`) })
+		.update({ special: knex.raw(`REPLACE(${specialKey}, 'geometry,', 'geometry.')`) })
```

Comment:
I believe knex has a helper for this in the .raw usage. Can you try if the following works as expected?

```suggestion
		.update({ special: knex.raw("REPLACE(??, 'geometry,', 'geometry.')", ['special']) })
```